platooning_control_ihp/src/platoon_control_ihp.cpp

```diff
@@ -262,7 +226,7 @@ namespace platoon_control_ihp
 
     void PlatoonControlIHPPlugin::generateControlSignals(const cav_msgs::TrajectoryPlanPoint& first_trajectory_point, const cav_msgs::TrajectoryPlanPoint& lookahead_point){
 
-        pcw_.setCurrentSpeed(trajectory_speed_); //TODO why this and not the actual vehicle speed?  Method name suggests different use than this.
```

Comment:
Are we sure this TODO has been addressed and is no longer relevant?  If things are okay as is, it would be good to add some clarity for future maintainers by either a) changing method name to reflect true meaning or b) add a comment explaining why it is being used this way.